### DISCUSSION TITLE
Improved and updated the i18n translation for German

### DISCUSSION THREAD

**Discussion by bannert1337** [open-webui/open-webui#3510] [Show and tell]
Hey,
I wanted to contribute to this project because I am very excited about it and use it every day.
My mother tongue is German and I have good English skills.
Sadly, I haven't used Svelte yet and only have basic knowledge of JavaScript and Python.
Therefore, I wanted to contribute in a “language” I am profound. :)

I made many changes to existing entries because they either were wrong, imprecise, or not very eloquent.
I also filled all missing entries.

**Comment by bannert1337** [comment]
I only used formal language ("Sie" instead of "du").
I could make another version with informal language ("Du" instead of "Sie").
Then the user can select formal or informal German.

**Comment by moblangeois** [comment]
@bannert1337 Thank you for your contribution to improving the German translations for Open WebUI!

I wanted to let you know that I've recently created a discussion about an automated translation improvement tool that might be of interest to you and other contributors working on localization. You can find it here: #3614 

Have a good day! :)

**Comment by nicedevil007** [comment]
Hi @bannert1337 

I already started translating the german file with the locales... I recently installed OpenWebUI and already saw a half translated LoginPage if you enabled SSO. But I stopped now as you are doing the same :) Hope it will be implemented very soon.

**Comment by alvarellos** [comment]
Hi Bannert, I have added Support for Multi Language for the Openwebui. It might be interesting for you.

Here is the discussion
[[Amber](https://github.com/amberyufangchiu)](https://github.com/open-webui/open-webui/discussions/15839)
and the PR
https://github.com/open-webui/open-webui/pull/15840

I would appreciate to have a feedback :)

Thanks !!
